docs/download.md

```diff
@@ -31,19 +31,24 @@ The plugin is tested on the following architectures:
 
 	NVIDIA Driver*: 470+
 
-	Python 3.6+, Scala 2.12, Java 8, Java 11, Java 17
+	Runtime: 
+		Scala 2.12
+		Python, Java Virtual Machine (JVM) compatible with your spark-version. 
+
+		* Check the Spark documentation for Python and Java version compatibility with your specific 
+		Spark version. For instance, visit `https://spark.apache.org/docs/3.4.1` for Spark 3.4.1. 
+		Please be aware that we do not currently support Spark builds with Scala 2.13.
```

Comment:
Minor nit, we already specify we do not support Scala 2.12, I do not think we need to call out what we do not support.  That is, I think this line can be removed.

Reply:
The idea is was to highlight that the plugin does not have special requirements for anything but Scala.